addons/l10n_fr_invoice_addr/models/account_move.py

```diff
@@ -0,0 +1,19 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_fr_is_company_french = fields.Boolean(compute='_compute_l10n_fr_is_company_french')
+
+    @api.depends('company_id.country_code')
+    def _compute_l10n_fr_is_company_french(self):
+        for record in self:
+            if record.company_id:
+                result = record.company_id.country_code in record.company_id._get_unalterable_country()
```

Comment:
record.l10n_fr ... = record.company_id.country_code in record.company_id._get_unalterable_country()